docs/design/keyboard-shortcuts.md

```diff
@@ -195,22 +195,59 @@ The complete schema for the shortcuts JSON is at [extended-manifest.schema.json]
 > [!NOTE]
 > Keytips, also known as sequential key shortcuts, such as the Excel shortcut to choose a fill color **Alt+H, H**, are not supported in Office Add-ins.
```

Comment:
```suggestion
> KeyTips, also known as sequential key shortcuts, such as the Excel shortcut to choose a fill color **Alt+H+H**, are not supported in Office Add-ins.
```

Reply:
> 

With sequential shortcuts, the comma is used, not another + sign.

Reply:
Thanks! I found this in the [Excel keyboard shortcut list](https://support.microsoft.com/en-us/topic/keyboard-shortcuts-in-excel-1798d9d5-842a-42b8-9c99-9b7213f0040f) but surprisingly, nowhere in the styleguide (that I could find, anyway.)